src/Confluent.Kafka/Producer.cs

```diff
@@ -1034,25 +1043,58 @@ public string Name
 
         /// <summary>
         ///     Asynchronously send a single message to the broker.
-        ///     Refer to <see cref="ProduceAsync{TKey, TValue}(string,TKey,TValue)" />
+        ///     Refer to <see cref="ProduceAsync(string, TKey, TValue, int, bool)" /> for more information.
         /// </summary>
         /// <remarks>
-        ///     The configured partitioner is used to determine partition message 
-        ///     is produced to.
+        ///     The partition the message is produced to is determined using the configured partitioner.
         /// </remarks>
         public Task<Message<TKey, TValue>> ProduceAsync(string topic, TKey key, TValue val, bool blockIfQueueFull)
-            => serializingProducer.ProduceAsync(topic, key, val, blockIfQueueFull);
-
-
+            => serializingProducer.ProduceAsync(topic, key, val, blockIfQueueFull);
+
+
+        /// <summary>
+        ///     Asynchronously send a single message to the broker (order of delivery reports strictly guarenteed).
```

Comment:
The summary is what gets displayed in intellisense - this note is there because I was thinking a user should see it in that. Ideally (ultimately) we don't want the see cref thing at all (see above note about including blocks).